rust/ql/test/library-tests/controlflow/CFG.ql

```diff
@@ -0,0 +1,13 @@
+/**
+ * @kind graph
```

Comment:
Rename to `Cfg.ql`. Also, I suggest removing `@kind graph` as, in my experience, it actually hurts more than it helps; for example, nodes with multiple edges between them get collapsed in this format